src/PDPVerifier.sol

```diff
@@ -11,12 +11,12 @@ import "../lib/openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgrad
 
 
 interface PDPListener {
-    function proofSetCreated(uint256 proofSetId, address creator) external;
-    function proofSetDeleted(uint256 proofSetId, uint256 deletedLeafCount) external;
-    function rootsAdded(uint256 proofSetId, uint256 firstAdded, PDPVerifier.RootData[] memory rootData) external;
-    function rootsScheduledRemove(uint256 proofSetId, uint256[] memory rootIds) external;
+    function proofSetCreated(uint256 proofSetId, address creator, bytes calldata extraData) external;
+    function proofSetDeleted(uint256 proofSetId, uint256 deletedLeafCount, bytes calldata extraData) external;
+    function rootsAdded(uint256 proofSetId, uint256 firstAdded, PDPVerifier.RootData[] memory rootData, bytes calldata extraData) external;
+    function rootsScheduledRemove(uint256 proofSetId, uint256[] memory rootIds, bytes calldata extraData) external;
     function posessionProven(uint256 proofSetId, uint256 challengedLeafCount, uint256 seed, uint256 challengeCount) external;
```

Comment:
Out of curiosity, why omit the extra data on this method? It will seem odd to a reader, so perhaps you could document it.